terraform/azure/main.tf

```diff
@@ -0,0 +1,63 @@
+terraform {
+  backend "remote" {}
+}
+
+provider "azurerm" {
+  version = "=2.25.0"
+
+  subscription_id = var.subscription_id
+  tenant_id       = var.tenant_id
+
+  features {}
+}
+
+module "service_cluster" {
+  source = "./modules/kubernetes-cluster"
+
+  prefix = var.prefix_sc == "" ? "${terraform.workspace}-service-cluster" : var.prefix_sc
+
+  machines = var.machines_sc
+
+  location = var.location
+
+  dns_suffix = "elastisys-azure.com"
```

Comment:
For manual testing you can set this to `elastisys.com` and add a NS record for the subdomain (in AWS) to azure.
Since we are moving this to a separate tool I thought this is not something to put effort into.

Reply:
Or just set individual addresses in your `/etc/hosts` to the ingress lb IP

Reply:
Ok, then for now we should either try to get a domain or document these workarounds

Reply:
I realized that we can just point `azure.elastisys.com` to azure NS. And it seems to work, so this might be a solution as well

Reply:
Then go for that :slightly_smiling_face:

Reply:
Might I suggest azure.compliantkubernetes.io instead? ;)

Reply:
Yea, sure. Is that the new go-to address for ck8s stuff?

Reply:
Sure, that's ok with me as well

Reply:
Don't know if it's the go-to address, but since we cleaned up all the Elastisys specific stuff before let's avoid introducing new things!